library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/BaseChuckerActivity.kt

```diff
@@ -30,4 +41,25 @@ internal abstract class BaseChuckerActivity : AppCompatActivity() {
     fun showToast(message: String) {
         Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
     }
+
+    /**
+     * Applies window insets padding to the provided view for Android 15+ devices.
+     * This handles system bars, display cutout, and IME insets.
+     */
+    private fun setWindowInsets(view: View) {
+        if (Build.VERSION.SDK_INT >= 35) {
```

Comment:
can put this to caller site, wont need to come to this func itself if build version is less 35